infrastructures/openshift/src/test/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftInternalRuntimeTest.java

```diff
@@ -145,6 +146,7 @@
   @Mock private OpenShiftEnvironmentProvisioner kubernetesEnvironmentProvisioner;
   @Mock private SidecarToolingProvisioner<OpenShiftEnvironment> toolingProvisioner;
   @Mock private UnrecoverablePodEventListenerFactory unrecoverablePodEventListenerFactory;
+  @Mock private OpenShiftExternalServerExposer serverExposerStrategy;
```

Comment:
I do not see where it's used. Please consider adapting tests to it, or removing the unused field.